tests/test-cases/Group9-VIC-Admin/9-1-VICAdmin-ShowHTML.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation  Test 9-1 - VICAdmin DisplayHTML
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  ${true}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+Default Tags  regression
```

Comment:
@mhagen-vmware Took out the "regression" field and added a specific entry to the "Regression Tests" to pull container logs and ensure the container's vmware.log is present.